samcli/lib/build/app_builder.py

```diff
@@ -322,8 +318,7 @@ def _parse_builder_response(stdout_data, image_name):
                 LOG.debug("Builder library does not support the supplied method")
                 raise UnsupportedBuilderLibraryVersionError(image_name, msg)
 
-            else:
-                LOG.debug("Builder crashed")
-                raise ValueError(msg)
+            LOG.debug("Builder crashed")
```

Comment:
Not blocking on this, but we have the same verbiage on the 'builder crashed' on if there was an exception or if there was error code we dont understand. We need to differentiate on that.